interface/globals.php

```diff
@@ -359,6 +359,12 @@ function undoMagicQuotes($array, $topLevel=true) {
 }
 $login_filler_line = ' bgcolor="#f7f0d5" ';
 $logocode = "<img src='$web_root/sites/" . $_SESSION['site_id'] . "/images/login_logo.gif'>";
+// optimal size for the tiny logo is height 43 width 86 px
+// inside the open emr they will be auto reduced
+$tinylogocode1 = "<img class='tinylogopng' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_1.png'>";
+$tinylogocode2 = "<img class='tinylogopng' src='$web_root/sites/" . $_SESSION['site_id'] . "/images/logo_2.png'>";
+$GLOBALS['tiny_logo_1']=false;
+$GLOBALS['tiny_logo_2']=false;
```

Comment:
Do you mean into library/globals.inc.php? because I do not see globals.php in the library folder